Sample Applications/DataBindingDemo/App.xaml

```diff
@@ -78,6 +78,23 @@
                         <Setter Property="BorderThickness" Value="3" TargetName="border" />
                         <Setter Property="Padding" Value="5" TargetName="border" />
                     </DataTrigger>
+                    <DataTrigger Binding="{Binding Path=(SystemParameters.HighContrast)}" Value="true" >
+                        <Setter TargetName="descriptionTitle" Property="TextBlock.Foreground" Value="{DynamicResource {x:Static SystemColors.WindowTextBrushKey}}"/>
+                        <Setter TargetName="currentPriceTitle" Property="TextBlock.Foreground" Value="{DynamicResource {x:Static SystemColors.WindowTextBrushKey}}"/>
```

Comment:
What about the MouseOVer case?  The ListBox is still drawing a selection adorner over that, you'd still need the highlights.